call-stack-to-nix.nix

```diff
@@ -0,0 +1,19 @@
+/* The function obtained when this is applied to a package set calls 
+ * the stack-to-nix tool on a supplied source set and then 
+ * imports the resulting pkgs.nix. The application of this function
+ * to a source path can thus be used directly as the input to mkStackPackageSet
+ */ 
+{ nix-tools, pkgs }: { src, stackYaml ? null }:
+let
+  pkgsNix = pkgs.stdenv.mkDerivation {
+    name = "pkgs-nix";
+    inherit src;
+    nativeBuildInputs = [ nix-tools pkgs.nix-prefetch-git pkgs.rsync ];
```

Comment:
why do we need `pkgs.rsync`

Reply:
`rsync` handles copying hidden files and directories (like `.plan.nix` and `.stack.nix`) better than `cp`. I couldn't get `cp` to do that reliably. Maybe there is a flag I am missing?

Reply:
Oh. I guess it is no longer used since I'm calling stack-to-nic directly into $out. I'll get rid of it.